src/game/default/g_main.c

```diff
@@ -938,6 +948,119 @@ void G_Shutdown(void) {
 	gi.FreeTag(MEM_TAG_GAME);
 }
 
+void G_CallTimeOut(g_entity_t *ent) {
+
+	g_level.match_status |= MSTAT_TIMEOUT;
```

Comment:
I would check if `g_timeout_time` is `0` and just return here. Some servers might want to disable calling timeout.